tests/test_utils.py

```diff
@@ -241,6 +263,44 @@ def test_spawn_multiprocessing_graph():
     mp.set_start_method("fork", force=True)
 
 
+def test_spawn_loky_graph_win():
+
+    if platform.system() == "Windows":
+        import loky
+    else:
+        return None
```

Comment:
nit-pick but always go for fast returns:

```suggestion
    if platform.system() != "Windows":
        return

    import loky
```